api/v1beta1/foundationdbcluster_types.go

```diff
@@ -2113,3 +2137,24 @@ const (
 	// PublicIPSourceService specifies that a pod gets its IP from a service.
 	PublicIPSourceService PublicIPSource = "service"
 )
+
+// ProcessClass models the different FDB classes
+type ProcessClass string
```

Comment:
Is this intended to be an exhaustive list, or just a list of process classes that have specialized behavior?

Reply:
In the end I think this could be an exhaustive list for now I only added the process classes that are used (in a follow up PR I would also adjust the tests to use these strings).